Uzoamaka126/My-Auto-Mart-Challenge
# #165906941 Implement login interface

#### What does this PR do?
Implement login interface

#### Description of Task to be completed?
- [x] Design and implementation of AutoMart login interface

#### How should this be manually tested?
* Clone https://github.com/Uzoamaka126/My-Auto-Mart-Challenge
* Visit `/login.html` on your browser
* You shoul see the image below

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#165906941

#### Screenshots (if appropriate)
![Screenshot (192)](https://user-images.githubusercontent.com/46677256/57585090-30fc1f00-74db-11e9-99ce-71fba4bfa0e0.png)

#### Questions:
None
